### DISCUSSION TITLE
hiding/disabling methods of crafting? (workstations)

### DISCUSSION THREAD

**Discussion by sippingmariowater** [shedaniel/RoughlyEnoughItems#985] [Q & A] [answered]
does REI have a function to hide crafting methods? hiding all recipes using a furnace, or a grinder, for example. there is a lot of clutter in my modpack due to workstations that arent even available/relevant.

**Comment by shedaniel** [comment, marked as answer]
I do not plan to add this as a feature, I think this could be exploitative.

**Reply by sippingmariowater** [reply]
what? how?
